### PR TITLE
Update alarm.dm

### PR DESCRIPTION
**Pull request by SirBloodBuske**
Fixes the ability to cut the wires in fire alarms and completely deconstruct them.

### CODE REVIEW

**Comment by M3hillus** [comment]
:+1:

**Comment by TheDZD** [comment]
Looks good, all set to be merged?

**Comment by SirBloodBuske** [comment]
Almost, Forgot one part.

File: code/game/machinery/alarm.dm [hunk 1]
```diff
@@ -1130,12 +1130,21 @@ FIRE ALARM
...
+				if(istype(W, /obj/item/device/multitool))
 					src.detecting = !( src.detecting )
 					if (src.detecting)
 						user.visible_message("\red [user] has reconnected [src]'s detecting unit!", "You have reconnected [src]'s detecting unit.")
 					else
-						user.visible_message("\red [user] has disconnected [src]'s detecting unit!", "You have disconnected [src]'s detecting unit.")
+						user.visible_message("\red [user] has disconnected [src]'s detecting unit!", "You have disconnected [src]'s detecting unit.")				
+				
+				else if(istype(W, /obj/item/weapon/wirecutters))  // cutting the wires out
+					user << "You cut the wires!"
```
Comment (Fox-McCloud) [inline code/game/machinery/alarm.dm]: spannsssss

Comment (SirBloodBuske) [inline code/game/machinery/alarm.dm]: MY bad, Ill go back and fix that!

Comment (SirBloodBuske) [inline code/game/machinery/alarm.dm]: src << "You cut the wires!" 

Is that right?

Comment (SirBloodBuske) [inline code/game/machinery/alarm.dm]: user << "<span class='warning'>You cut the wires!</span>"

**Comment by SirBloodBuske** [comment]
Alright I fixed the spans like fox asked me too!

**Comment by Fox-McCloud** [comment]
@SirBloodBuske 

Sorry for not answering your question in the ahelp regarding this--was busy at the time; looks like you figured it out though.

**Comment by SirBloodBuske** [comment]
It's okay, I am learning and I also learned that there is a lot of old code that does not have the spanners added so I'll go through all of it and add some and put it up here!

**Pull request merged**